src/main/java/com/scalar/cassy/service/AzureBlobContainerClientModule.java

```diff
@@ -0,0 +1,37 @@
+package com.scalar.cassy.service;
+
+import com.azure.storage.blob.BlobContainerAsyncClient;
+import com.azure.storage.blob.BlobServiceAsyncClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+
+public class AzureBlobContainerClientModule extends AbstractModule {
```

Comment:
I think it's slightly better to make this class abstract and make `AzureBlobBackupModule` and `AzureBlobRestoreModule` extend this so that users of modules don't need to create injector from 2 modules like [this](https://github.com/scalar-labs/cassy/pull/41/files#diff-cd17c62ec90758e5318ce3283237dd68R53).

Thought ?

Reply:
Thank you. 
I agree, it will be more clean.